hw-model/src/lib.rs

```diff
@@ -919,6 +956,18 @@ mod tests {
         model.soc_ifc().cptra_fuse_wr_done().write(|w| w.done(true));
         model.soc_ifc().cptra_bootfsm_go().write(|w| w.go(true));
 
+        // Set up the PAUSER as valid for the mailbox (using index 0)
+        model
+            .soc_ifc()
+            .cptra_mbox_valid_pauser()
+            .at(0)
+            .write(|_| 0x1);
```

Comment:
Should this be using the `valid_pauser` value from BootParams?

Reply:
Good question, so these tests use new_unbooted and because of that, they don't have BootParams. Based on other setup I thought "new" was the right place for this. But I'm open to input on that.